package-examples/function-export/resources/resources.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+  namespace: development
+  labels:
+    app: nginx
+    # You can remove this label and see what will happen to your workflow result.
+    owner: alice
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+        - name: nginx
+          image: nginx:1.14.2
+          ports:
+            - containerPort: 80
+---
+apiVersion: v1
```

Comment:
Let's move the namespace to be the first object. It won't affect anything though. It is consistent with the order that they should be applied.